src/Main.elm

```diff
@@ -200,90 +193,23 @@ pageView model siteMetadata page viewForPage =
             }
 
 
-articleImageView : ImagePath Pages.PathKey -> Element msg
-articleImageView articleImage =
-    Element.image [ Element.width Element.fill ]
-        { src = ImagePath.toString articleImage
-        , description = "Article cover photo"
-        }
-
-
-header : PagePath Pages.PathKey -> Element msg
-header currentPath =
-    Element.column [ Element.width Element.fill ]
-        [ Element.el
-            [ Element.height (Element.px 4)
-            , Element.width Element.fill
-            , Element.Background.gradient
-                { angle = 0.2
-                , steps =
-                    [ Element.rgb255 0 242 96
-                    , Element.rgb255 5 117 230
-                    ]
-                }
-            ]
-            Element.none
-        , Element.row
-            [ Element.paddingXY 25 4
-            , Element.spaceEvenly
-            , Element.width Element.fill
-            , Element.Region.navigation
-            , Element.Border.widthEach { bottom = 1, left = 0, right = 0, top = 0 }
-            , Element.Border.color (Element.rgba255 40 80 40 0.4)
-            ]
-            [ Element.link []
-                { url = "/"
-                , label =
-                    Element.row [ Font.size 30, Element.spacing 16 ]
-                        [ DocumentSvg.view
-                        , Element.text "elm-pages-starter"
-                        ]
-                }
-            , Element.row [ Element.spacing 15 ]
-                [ elmDocsLink
-                , githubRepoLink
-                , highlightableLink currentPath pages.blog.directory "Blog"
-                ]
-            ]
-        ]
-
-
-highlightableLink :
-    PagePath Pages.PathKey
-    -> Directory Pages.PathKey Directory.WithIndex
-    -> String
-    -> Element msg
-highlightableLink currentPath linkDirectory displayName =
-    let
-        isHighlighted =
-            currentPath |> Directory.includes linkDirectory
-    in
-    Element.link
-        (if isHighlighted then
-            [ Font.underline
-            , Font.color Palette.color.primary
-            ]
-
-         else
-            []
-        )
-        { url = linkDirectory |> Directory.indexPath |> PagePath.toString
-        , label = Element.text displayName
-        }
-
-
 commonHeadTags : List (Head.Tag Pages.PathKey)
 commonHeadTags =
     [ Head.rssLink "/blog/feed.xml"
     , Head.sitemapLink "/sitemap.xml"
     ]
 
 
-{-| <https://developer.twitter.com/en/docs/tweets/optimize-with-cards/overview/abouts-cards>
-<https://htmlhead.dev>
-<https://html.spec.whatwg.org/multipage/semantics.html#standard-metadata-names>
-<https://ogp.me/>
+
+{- Read more about the metadata specs:
```

Comment:
Shouldn't this have been `{-|`?
Now this documentation isn't attached to `head` anymore

Reply:
I changed it from doc to comment because it doesn’t explain how to use head
or what year does, it provides background information that might be useful
to a developer wanting to change head. I guess it would make more sense
inside head now than just before it?